backend/src/main/scala/bloop/Compiler.scala

```diff
@@ -357,13 +366,74 @@ object Compiler {
 
     val uniqueInputs = compileInputs.uniqueInputs
     reporter.reportStartCompilation(previousProblems)
+    val fileManager = newFileManager
+
+    // Manually skip redundant best-effort compilations. This is necessary because compiler
+    // phases supplying the data needed to skip compilations in zinc remain unimplemented for now.
```

Comment:
Yes, I tried to enable those phases, but it would have taken way too much time to do it properly (without excessive crashing). I planned to go back and start implementing that after the initial merges (and I would still love to do that), but to be completely transparent I do not know if that will be possible (time-wise).

Reply:
ach, right. Could we make sure there is an issue about it in the compiler?

Reply:
Done: https://github.com/scala/scala3/issues/20704

Reply:
Thanks!